docs/deploy/linux/installer/install.md

```diff
@@ -173,9 +173,9 @@ the installer script will install packages in the single-version layout.
 For the multi-version ROCm installation you must use the installer script from
 the latest release of ROCm that you wish to install.
 
-**Example:** If you want to install ROCm releases 5.3.3 and 5.5.1
+**Example:** If you want to install ROCm releases 5.3.3 and 5.4.3
 simultaneously, you are required to download the installer from the latest ROCm
-release v5.5.1.
+release v5.4.3.
```

Comment:
Should this be 5.6?
We should stop using "v" prior to version numbers. It is unnecessary.

Reply:
Keeping for consitency